docs/commands/New-Fixture.mdx

```diff
@@ -51,33 +49,33 @@ $PSCommandPath.Replace('.Tests.ps1', '.ps1')
 Describe "Clean" {
 
     It "Returns expected output" {
-        Clean | Should -Be $true
+        Clean | Should -Be "YOUR_EXPECTED_VALUE"
     }
 }
-```
+```powershell
```

Comment:
Where did `powershell` come from? Not in source

Reply:
That is coming from the Alt3 module. Missed it but is happening in my test suite as well. 

Good catch!